flow/platforms/asap7/config.mk

```diff
@@ -81,7 +81,7 @@ export DONT_USE_CELLS          += SDF* ICG*
 export LATCH_MAP_FILE          = $(PLATFORM_DIR)/yoSys/cells_latch_R.v
 export CLKGATE_MAP_FILE        = $(PLATFORM_DIR)/yoSys/cells_clkgate_R.v
 export ADDER_MAP_FILE         ?= $(PLATFORM_DIR)/yoSys/cells_adders_R.v
-export MAX_UNGROUP_SIZE       ?= 100
+export MAX_UNGROUP_SIZE       ?= 1000
```

Comment:
Is this due to a difference in the estimated area?  Why only in asap7?

Reply:
asap7 is one of those which moved the most, `MAX_UNGROUP_SIZE` is now in number of nand2 gates, not in platform's area units

Reply:
That sounds like it would affect all platforms.  Is a doc update needed as well?

Reply:
Yes, it is